sphinx-doc/requirements.txt

```diff
@@ -1,3 +1,4 @@
+h5py
```

Comment:
```suggestion
```
With my suggested change to the `try / except ImportError` above, this is no longer needed.